py-polars/requirements-dev.txt

```
@@ -38,7 +38,7 @@ cloudpickle
 fsspec
 s3fs[boto3]
 # Spreadsheet
-fastexcel>=0.9
+fastexcel>=0.11
```

```suggestion
fastexcel>=0.11.1
```
@lukapeschke https://github.com/ToucanToco/fastexcel/pull/261